win32/src/gballoc_ll_jemalloc.c

```diff
@@ -214,3 +218,97 @@ void gballoc_ll_print_stats(void)
     /* Codes_SRS_GBALLOC_LL_JEMALLOC_01_008: [ gballoc_ll_print_stats shall call je_malloc_stats_print and pass to it jemalloc_print_stats_callback as print callback. ]*/
     je_malloc_stats_print(jemalloc_print_stats_callback, NULL, NULL);
 }
+
+int gballoc_ll_set_decay(int64_t decay_milliseconds)
+{
+    int result = 0;
+
+    /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_001: [ gballoc_ll_set_decay shall advance jemalloc's internal epoch counter. ]*/
+    if (je_mallctl("epoch", NULL, NULL, NULL, 0) != 0)
```

Comment:
>je_mallctl("epoch", NULL, NULL, NULL, 0) [](http://example.com/codeflow?start=8&length=40)

errr... why do we need to set the epoch?

I've been reading the documentation and it seems to be that the epoch is used for reporting statistics.

`Take special note of the epoch mallctl, which controls refreshing of cached dynamic statistics.`

Reply:
I have removed the epoch call since we would not be making use of the stats here.